js/websocket.js

```diff
@@ -0,0 +1,14 @@
+if (window.location.hostname.indexOf("lndo") > 0) {
```

Comment:
indexOf gives -1 when not found, 0 when the strings starts with substring.
So best practice is "> -1"